documentation/style-guide.md

```diff
@@ -0,0 +1,249 @@
+## Links
+
+- Use Markdown syntax for links: `(text)[url]`.
+- Do not link to parent concept or topic entries. The Docs navigation provides those for each entry.
+- Do not link out to external third-party documentation. (Such as Wikipedia.)
+- You are encouraged to link out to other relevant Docs topics, concepts, or terms the first time they appear.
+- In some cases, it may be appropriate to link out to external standards documents or entities. (Example: [XML 1.0 Specification](http://www.w3.org/TR/REC-xml), [World Wide Web Consortium](https://www.w3.org/standards/), [Red Hat Enterprise Linux](https://www.redhat.com/en/technologies/linux-platforms/enterprise-linux))
+- When linking items outside the document, use the full path. When linking to items inside the document (i.e. Headers) use a relative path.
+- If there are parentheses in the link, replace them with `%28` and `%29`.
+
+## File Hosting
+
+- If you'd like to host files such as images, or GIFs, please upload it to the [/media folder](https://github.com/Codecademy/docs/tree/main/media).
+- Images should be linked to using the following url: "https://raw.githubusercontent.com/Codecademy/docs/main/media/" followed by the image name.
+- File names should use dashes (`-`) to separate words rather than spaces or underscores.
+- File names should be in lower case.
+- Images should be used for diagrams, UI illustrations, and showing output to the user.
+- Images should not be used to illustrate code. [Code blocks](#code) should be used instead.
+
+> **Note:** Image links will not work until a PR is merged.
+
+> **Note:** Some existing images may use `https://static-assets.codecademy.com/Courses/` in their links. These come from existing Codecademy course content and are for internal use only. Please disregard those and use the above format instead.
+
+## Metadata
+
+Each Docs entry is preceded by metadata that provides information about how the entry is displayed.
+
+The metadata for a concept or term entry should look like this:
+
+```
+---
+Title: 'The Title'
+Description: 'A brief description'
+Subjects:
+  - 'A subject name'
+  - 'A second subject name'
+  - 'An nth subject name'
+Tags:
+  - 'A tag'
+  - 'A second tag'
+  - 'An nth tag'
+CatalogContent:
+  - 'learn-example-course'
+  - 'path/example-path'
+---
+```
+
+> **Note:** The template above does not apply to topic entries. See our [topic entry template](https://github.com/Codecademy/docs/blob/main/documentation/topic-info-template.md) for more details.
+
+- The metadata starts on the first line of the entry.
+- The metadata begins and ends with three dashes (`---`) on their own line.
+- There should be a line-break after the last line of the metadata.
+- There should not be extra line breaks within the metadata.
+- The data for `Title:` and `Description:` appear as single-quoted strings on the same line.
+- Do not use apostrophes or quotes in the title or description.
+- Do not use boldface or italics in the title or description.
```

Comment:
This seems to be in opposition to what I've seen in some of our content. I think I generally see the term in bold if it is used explicitly in the description. Perhaps we should include some additional context here to draw that distinction.

Reply:
On line 47 I do see that the first occurrence of the term should be bolded. I think this section is only about the metadata but I can see how this can be confusing. I'm wondering what else you think we can do to make this more explicit 🤔

Reply:
I changed it to "`Title:` and `Description:`" so it's clearer that it refers to the metadata entry.